package.json

```diff
@@ -3,12 +3,13 @@
   "private": true,
   "license": "MIT",
   "scripts": {
-    "test": "mocha -b $(find ./challenges -name test.js)",
-    "test:watch": "mocha -bw $(find ./challenges -name test.js)",
+    "format": "prettier --write ./**/*.js",
     "lint": "prettier --check ./**/*.js",
-    "format": "prettier --write ./**/*.js"
+    "test": "jest",
```

Comment:
Is it possible to make Jest generate a verbose output? I'm looking into the build output and it doesn't show any information about tests that ran during the process:

```
$ npm test
> challenges@ test /home/travis/build/dsafio/challenges
> jest
 PASS  challenges/fizz-buzz/test.js
 PASS  challenges/hello-world/test.js
```

Maybe the `test` could leverage Jest's `--verbose` flag. 🙂

Reply:
Ok! 😄